bom/pom.xml

```diff
@@ -458,11 +454,6 @@
                 <groupId>software.amazon.awssdk</groupId>
                 <version>${awsjavasdk.version}</version>
             </dependency>
-            <dependency>
```

Comment:
Is this part of the change?

Reply:
No, it just felt weird that we were specifying a dependency on the code generator in our BOM. I was hoping you'd ask! Why is this here?

Reply:
If you depended directly on the code generator you'd have to manually specify the version. Just seems natural to include all modules that are versioned together in the BOM since it's just a giant manifest of versions that were built and compatible with one another.

Reply:
I wasn't sure why customers would depend on the code generator. I'd think it would be excluded, similar reasons to why the codegen-maven-plugin is excluded.

Reply:
It's unlikely but customers may be doing something meta that requires it. There just doesn't seem to be a compelling reason to exclude it since it causes no harm. It's more likely to be used by us in other vended products that depend on the SDK.

Reply:
Should we include the maven plugin, as well?

Reply:
yes - I'd say we should include everything we vend.